### ISSUE TITLE
Combobox Not Announcing "No Results" status (NVDA)

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1201]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/c5cc4b1c-5886-11ee-9399-9336eb8a0144?sortField=ordinal&sortDir=asc&filter%5Brule%5D=manual&filter%5Bpage_number%5D=49&row=0

Status message is not automatically announced by the screen reader.
Element description: The status message "No results found".
Element location: This applies to each combo box in the examples section.

Steps to reproduce: Type an option that is not present in the combo box. The status message appears but is not announced automatically.
